src/vs/platform/list/browser/listService.ts

```diff
@@ -1139,6 +1140,12 @@ configurationRegistry.registerConfiguration({
 			'default': false,
 			'description': localize('horizontalScrolling setting', "Controls whether lists and trees support horizontal scrolling in the workbench.")
```

Comment:
I think that the description of the `horizontalScrollingKey` setting should explain the situation that it's possible to override this setting for SCM by setting `horizontalScrollingSCMKey`, which inherits this setting by default.